ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -8,12 +8,8 @@
 
 /* Begin PBXBuildFile section */
 		1498D2341E8E89220040F4C2 /* GeneratedPluginRegistrant.m in Sources */ = {isa = PBXBuildFile; fileRef = 1498D2331E8E89220040F4C2 /* GeneratedPluginRegistrant.m */; };
-		2D5378261FAA1A9400D5DBA9 /* flutter_assets in Resources */ = {isa = PBXBuildFile; fileRef = 2D5378251FAA1A9400D5DBA9 /* flutter_assets */; };
```

Comment:
Did you test this application on an iOS device?

Reply:
Yes I ran first in an iOS emulator. That's why you see all the changes in ios/ folder. 

I guess I should have removed them from git, My bad. But it does no harm.

Reply:
No need to remove these files